source/index.txt

```diff
@@ -1,10 +1,51 @@
+:noprevnext:
+
 =========================
 {+atlas-arch-center+}
 =========================
 
 .. default-domain:: mongodb
 
-Landing page content
+The {+atlas-arch-center+} documents the best way to build a robust data
+platform in {+service+}. While the :atlas:`{+service+} documentation 
+</>` provides information about how to use {+service+}, the 
+{+atlas-arch-center+} gives recommendations on the exact 
+configurations you should use to optimize governance, security,
+automation, and more.
+
+The {+atlas-arch-center+}'s content aligns with the {+waf+} pillars:
+
+.. image:: /includes/images/landing-image.png
+   :alt: "An image representing MongoDB Atlas supported by the Atlas Well-Architected Framework pillars."
+   :width: 500px
+   :align: center
+
+
+- **Operational Efficiency**: Automation, monitoring, and observability
```

Comment:
Question/Suggestion: Should each pillar be linked to it's tab? I thought of this as I read for users who might not be fully new to Atlas, but are looking to read about one specific pillar. I see that you do mention the navigation in the section right below, but do users always read to the very bottom of a page?

Reply:
So the pillar pages in docs will be drawer pages only- the info on each pillar will be on the Atlas Well-Architected Framework website linked above the diagram. I could link again from each pillar name, but it won't go directly to the relevant pillar unless there are page anchors built in, they would all go to the same page, which isn't great.

Reply:
Sarah, related to this comment, on the left nav, is it possible to have a landing page or at least a link for the high-level pillars ( security, operational efficiency, etc.)? Because from the web page, we will be giving them a link for each pillar and I would like for it to link to these specific pages in docs -- for example the security ( learn more) tab on the web page should link to the security high level page or link on docs